subsys/bluetooth/controller/hci_driver.c

```diff
@@ -640,6 +640,8 @@ static int configure_memory_usage(void)
 
 	if (IS_ENABLED(CONFIG_BT_PER_ADV_SYNC)) {
 		cfg.periodic_sync_count.count = SDC_PERIODIC_ADV_SYNC_COUNT;
+		cfg.periodic_sync_buffer_cfg.count = CONFIG_BT_CTLR_SDC_PERIODIC_SYNC_BUFFER_COUNT;
+
```

Comment:
This does not work. Setting the sync count and buffer configurations must be done in separate steps as cfg is a union

Reply:
Yes, I just saw it was causing mem issue. pushed the fix separating them.